src/python/detectors/pytorch_create_tensors_directly_on_device/pytorch_create_tensors_directly_on_device.py

```diff
@@ -0,0 +1,18 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=pytorch_create_tensors_directly_on_device@v1.0 defects=1}
```

Comment:
The rule id doesn't look correct. Shouldn't it be `pytorch-create-tensors-directly-on-device@v1.0`?

with `-` and not `_`?